auth/templates.go

```diff
@@ -3,4 +3,6 @@ package auth
 const inviteUserTemplate = `
 <b>Welcome to Mission Control</b>
 <br><br>
-Hello %s, use this link for signing up: %s`
+Hello %s,
+
+please visit <a href="%s">this link</a> to complete registration and use the code: <code>%s</code>`
```

Comment:
Can we switch this to a gotemplate ?

```suggestion
Please visit <a href="%s">%s</a> to complete registration and use the code: <code>%s</code>`
```